language-support/java/bindings-rxjava/src/test/scala/com/daml/ledger/rxjava/grpc/helpers/LedgerServices.scala

```diff
@@ -189,12 +188,12 @@ final class LedgerServices(val ledgerId: String) {
     }
   }
 
-  def withTransactionClient(
+  def withTransactionsClient(
```

Comment:
We have a lot more `transactionClient` (singular) than `transactionsClient` (plural) in our code base. The java world seems to be using the plural form, is that why you renamed it?

Reply:
Yes, keeping consistency with the base library. The naming is wrong, but I prefer consistency over correctness. I don't have a strong opinion on this one in particular, so I'll happily revert this if you think it's worth it. :slightly_smiling_face:

Reply:
No, consistency is good.

Reply:
Cool, thanks. :slightly_smiling_face: